hs-bindgen/fixtures/documentation/doxygen_docs/bindingspec.yaml

```diff
@@ -40,9 +37,6 @@ ctypes:
 - headers: documentation/doxygen_docs.h
   cname: processor_fn_t
   hsname: Processor_fn_t
-- headers: documentation/doxygen_docs.h
-  cname: processor_fn_t_Deref
-  hsname: Processor_fn_t_Deref
```

Comment:
@TravisCardwell This is an auxiliary type that we generate, with no corresponding C type at all. I've opened https://github.com/well-typed/hs-bindgen/issues/1379 to think about cases like this.